elfi/diagnostics.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import scipy.stats as ss
+
+from elfi.utils import compare
+
+
+def l1_dist(estimated, reference, spec, method='pdf'):
+    """Compute the l1-distance between an estimation and a reference.
+    
+    Parameters
+    ----------
+    estimated :
+        an object to compare
```

Comment:
Any objects that have the same method. This method is evaluated at each point of the specified grid and the L1 distance is computed between the resulting vectors.